### DISCUSSION TITLE
Change Strip Led preset by action

### DISCUSSION THREAD

**Discussion by barbatrukko** [sbidy/wiz_light#184] [Q&A]
Hi, I have 2 stripled Wiz, and i have 2 Shelly Button1 (Shelly Button1 can simulate 4 states by mqtt)
In wiz app I can have 4 preset and also more effect.
I would use by Shelly Button for use multiple preset: for example
1 click -> Open to Solar light or toggle off
2 click  -> Open with Immersion preset
3 click -> Open XXX preset
Long Click -> open YYY preset.
How I can change effect, intensity and speed by automation or action?

**Comment by sjee** [comment]
You may also want to ask here: https://www.shelly-support.eu/forum/index.php?board/101-shelly-button-1/
